hack/tool/cmd/aws/aws.go

```diff
@@ -1,18 +1,6 @@
-/*
-Copyright 2022 k0s authors
+// SPDX-FileCopyrightText: 2022 k0s authors
+// SPDX-License-Identifier: Apache-2.0
 
```

Comment:
Adding an additional blank line here between the headers and the package declaration, as this is not a package comment. Same for some other Go files in the hack/tool folder.